File: src/main/java/com/wangkaihua/myblog/service/BlogService.java
package com.wangkaihua.myblog.service;

import com.wangkaihua.myblog.common.PageResult;
import com.wangkaihua.myblog.entity.TbBlog;
import com.wangkaihua.myblog.entity.vo.BlogVO;

import java.util.List;

/**
 * @author wangkaihua
 * @create 2018-08-29 21:55
 **/
public interface BlogService {

    List<TbBlog> findAll();

    void add(BlogVO blogVO);

    void delete(int[] ids);

    BlogVO findOne(int id);

    PageResult findPage(int pageNum, int pageSize);

    void update(BlogVO blogVO);
}

File: src/main/webapp/js/service/blogService.js
/**
 * Created by hua on 2018/9/5.
 */
app.service('blogService',function ($http) {
    this.add = function (entity) {
        return $http.post("/blog/add",entity);
    }

    this.findAll = function () {
        return $http.get("/blog/findAll");
    }

    this.findPage = function (pageNum,pageSize) {
        return $http.get("/blog/findPage?pageNum="+pageNum+"&pageSize="+pageSize);
    }

    this.delete = function (ids) {
        return $http.get("/blog/delete?ids="+ids);
    }

    this.findOne = function (id) {
        return $http.get("/blog/findOne?id="+id);
    }
})